frontend/src/app/components/skeletons/placeholder-table-data/placeholder-table-data.component.html

```
@@ -1,5 +1,14 @@
-<div class="mat-elevation-z4 data-table">
+<div class="mat-elevation-z4 data-table data-table_desktop">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)